tools/ci_build/build.py

```diff
@@ -14,118 +14,41 @@
 import sys
 from pathlib import Path
 
-
-def version_to_tuple(version: str) -> tuple:
-    v = []
-    for s in version.split("."):
-        with contextlib.suppress(ValueError):
-            v.append(int(s))
-    return tuple(v)
-
-
 SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
 REPO_DIR = os.path.normpath(os.path.join(SCRIPT_DIR, "..", ".."))
 
 sys.path.insert(0, os.path.join(REPO_DIR, "tools", "python"))
```

Comment:
Modifying sys.path is usually not the best thing to do. Given a chance I would find ways to remove it.